Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /root
+ENV VENV /opt/venv
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONPATH /root
+
+# This is necessary for opencv to work
+RUN apt-get update && apt-get install -y libsm6 libxext6 libxrender-dev ffmpeg build-essential
+
+# Install the AWS cli separately to prevent issues with boto being written over
+RUN pip3 install awscli
```

Comment:
can we add a comment here to install gsutil for Google

Reply:
done

Reply:
Lets actually have the install command - https://github.com/flyteorg/flytesnacks/blob/master/legacy/python/Dockerfile#L15-L17

Reply:
commented

Reply:
done